js/base/Exchange.js

```diff
@@ -2196,6 +2196,11 @@ module.exports = class Exchange {
         return '1e' + Precise.stringNeg (precision)
     }
 
+    safeTicksizeFromDigit (object, key) {
```

Comment:
This naming is not entirely correct, we can't extract the tick size from a digit. We can only extract it from a number or from a string. I would refrain from adding this method for now.